packages/cli/src/controllers/tags.controller.ts

```diff
@@ -14,41 +27,46 @@ export class TagsController {
 
 	// TODO: move this into a new decorator `@IfEnabled('workflowTagsDisabled')`
 	@Middleware()
-	workflowsEnabledMiddleware(_req: Request, _res: Response, next: NextFunction) {
-		if (this.config.getEnv('workflowTagsDisabled'))
-			throw new BadRequestError('Workflow tags are disabled');
+	workflowsEnabledMiddleware(_req: Request, res: Response, next: NextFunction) {
+		if (this.config.getEnv('workflowTagsDisabled')) {
```

Comment:
Throwing an exception from a middleware returns 500 with an empty response instead of the BadRequest. Probably a bug, so for now use the response object to return.